chia/consensus/block_creation.py

```diff
@@ -165,15 +161,16 @@ def create_foliage(
     foliage_transaction_block_hash: Optional[bytes32]
 
     if is_transaction_block:
-        cost = uint64(0)
+        cost: uint64
```

Comment:
I get a bit uneasy with uninitialized values - would it make sense to set these to `0` and reassign, so the `else` can be removed?